socorro/cron/mixins.py

```diff
@@ -90,71 +92,12 @@ def both_inits(self, *args, **kwargs):
 
 using_postgres = partial(
     with_transactional_resource,
-    'socorro.cron.connection_factory.ConnectionFactory',
+    'socorro.external.postgresql.connection_context.ConnectionContext',
```

Comment:
I nixed `ConnectionFactory` since it was a duplicate of `ConnectionContext` from when I vendored crontabber.